_posts/2016-02-07-the-error-model.md

```diff
@@ -458,9 +458,9 @@ too: these are failures that are triggered by hardware faults, like access viola
 however.  An arbitrary thread can inject a failure at nearly any point in your code.  Even between the RHS and LHS of an
 assignment!  As a result, things that look atomic in source code aren't.  [I wrote about this 10 years ago](
 http://joeduffyblog.com/2005/03/18/atomicity-and-asynchronous-exception-failures/) and the challenges still exist,
-although the risk has lessened as the .NET generally learned that thread aborts are problematic.  The new CoreCLR even
-lacks AppDomains, and the new ASP.NET 5 stack certainly doesn't use thread aborts like it used to.  But the [APIs are
-still there](https://github.com/dotnet/coreclr/blob/master/src/mscorlib/src/System/Threading/Thread.cs#L517).
+although the risk has lessened as .NET generally learned that thread aborts are problematic.  The new CoreCLR even
+lacks AppDomains, and the new ASP.NET Core 1.0 stack certainly doesn't use thread aborts like it used to.  But the [APIs are
+still there](https://github.com/dotnet/coreclr/blob/99e7f7c741a847454ab0ace1febd911378dcb464/src/mscorlib/src/System/Threading/Thread.cs#L518).
```

Comment:
A permalink to the sha is more future proof.

Btw. while the APIs are still there in the source code, they aren't exposed in .NET Core so I'm not sure this is a good example.